apps/wallet-mobile/src/components/Analytics/Analytics.tsx

```diff
@@ -235,9 +236,13 @@ const useStyles = () => {
       fontSize: 16,
     },
     toggle: {
-      padding: 16,
       flexDirection: 'row',
       alignItems: 'center',
+      fontSize: 24,
+    },
+    toggle_text: {
+      ...atoms.body_1_lg_medium,
+      fontWeight: '500',
```

Comment:
this is one of many things that is not good right now, the _lg_medium should have `fontWeight: 500` over there, but is just using `Rubik-Medium` we should move to `TT Rubik`